Day-II-Calculator/calculator/src/components/ButtonComponents/ActionButton.js

```diff
@@ -1,2 +1,13 @@
 import React from 'react';
-import './Button.css';
+
+const ActionButton = props => {
+    return (
+        <>
+            <button className="action-button" value={props.text}>
+                {props.text}
```

Comment:
This looks like a good opportunity for [object destructuring](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment#Object_destructuring)! For example:
```javascript
const ActionButton = ({ text }) => {
  return (
    <button className="action-button" value={text}>
      {text}
    </button>
  );
};
```